access/resource_secret_scope.go

```diff
@@ -21,10 +21,14 @@ type SecretScopesAPI struct {
 
 // Create creates a new secret scope
 func (a SecretScopesAPI) Create(scope string, initialManagePrincipal string) error {
-	return a.C.Post("/secrets/scopes/create", map[string]string{
-		"scope":                    scope,
-		"initial_manage_principal": initialManagePrincipal,
-	}, nil)
+	paramsMap := map[string]string{
+		"scope": scope,
```

Comment:
if `initialManagePrincipal`, isn't it omitting it on marshaller side?..

Reply:
it's omitted only when used in struct fields, annotated with `omitempty`

Reply:
Thanks for confirming. For GET requests, where map turns into a query string - it’s omitted, just in case.